formatters/lcov/lcov.go

```diff
@@ -64,7 +65,6 @@ func (r Formatter) Format() (formatters.Report, error) {
 			}
 			for ln-curLine >= 1 {
 				sf.Coverage = append(sf.Coverage, formatters.NullInt{})
-				sf.CalcLineCounts()
```

Comment:
@ale7714 can you check my logic that this can be removed? Justification is in 3908eec. From what I can see there's a few other formatters also calling this method without need.

Reply:
good catcht! looks good to me